6/Dockerfile.c9s

```diff
@@ -17,10 +17,8 @@ LABEL summary="$SUMMARY" \
       io.k8s.display-name="Varnish 6" \
       io.openshift.expose-services="8080:http,8443:https" \
       io.openshift.tags="builder,varnish6,varnish-6" \
-      com.redhat.component="varnish-6-container" \
-      com.redhat.license_terms="https://www.redhat.com/en/about/red-hat-end-user-license-agreements#rhel" \
```

Comment:
Could this be moved to at least a different commit, if not a different PR, stating, that the changes are in c9s, not c8s?

Reply:
Everything is generated by dist-gen, I will try to reorder commit.

Reply:
The general idea in the case of dist-gen is to update distgen files as one commit and add a second commit with generated files.